sklego/preprocessing/identitytransformer.py

```diff
@@ -9,16 +9,27 @@ class IdentityTransformer(BaseEstimator, TransformerMixin):
     The identity transformer returns what it is fed. Does not apply anything useful.
     The reason for having it is because you can build more expressive pipelines.
     """
+    def __init__(
+        self,
+        check_X: bool = True
```

Comment:
When I think about what the identitytransformer is supposed to do, I think `False` would be a better default here.

Reply:
I thought setting it to True here would be better for backward compatibility, since older versions are checking input array

Reply:
I see the reasoning, but pragmatically the `IdentityTransformer` is mainly used for no-op use-cases. So it feels fine to omit the `copy`. Unless @MBrouns you see a good reason to keep it?